pkg/codegen/dotnet/utilities.go

```diff
@@ -129,6 +129,12 @@ func getHelperMethodIfNeeded(functionName string) (string, bool) {
 		return `private static string ReadFileBase64(string path) {
 		return Convert.ToBase64String(System.Text.UTF8.GetBytes(File.ReadAllText(path)))
 	}`, true
+	case "filebase64sha256":
+		return `private static string ComputeFileBase64Sha256(string path) {
+		var fileData = System.Text.UTF8.GetBytes(File.ReadAllText(path))
+		var hashData = SHA256.Create().ComputeHash(fileData)
+		return Convert.ToBase64String(BitConverter.ToString(hashData).Replace("-","").ToLowerInvariant())
```

Comment:
This is different from `sha1` below in that it converts the result of `BitConverter.ToString` to base64. Is that intentional? It would be helpful to include a description of the behavior of `filebase64sha256` in the PR comment (and perhaps a link to the source docs).

Reply:
You were correct, there was a mistake. `filebase64sha256` is supposed to take in a file, convert it into a hash, and then convert those hash bytes into a Base64 string.